LibVLCSharp/Shared/Events/MediaEventManager.cs

```diff
@@ -1,208 +1,98 @@
 ﻿using System;
-using LibVLCSharp.Shared.Helpers;
 
 namespace LibVLCSharp.Shared
 {
     internal class MediaEventManager : EventManager
     {
         readonly object _lock = new object();
```

Comment:
You can remove this as well I guess. You think the locks aren't needed anymore? Not sure they ever were

Reply:
indeed, the lock has been pushed down to the attach/detach method